test/L2/paused/L2GovernorPaused.t.sol

```diff
@@ -28,6 +32,17 @@ contract L2GovernorPausedTest is Test {
     TimelockController timelock;
     address initialOwner;
 
+    function assetInitParamsEq() internal {
```

Comment:
it should be called `assertInitParamsEq`

Reply:
updated